Stripe/STPPaymentMethodsInternalViewController.m

```diff
@@ -134,8 +135,13 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
             [self.navigationController pushViewController:paymentCardViewController animated:YES];
         }
         else {
-            // Go to Add Source VC
-            // TODO: Link to Add Source VC here
+            STPAddSourceViewController *addSourceViewController = [[STPAddSourceViewController alloc] initWithSourceType:paymentType.sourceType
```

Comment:
this is a nullable initializer, so we should probably check for nil before continuing.